adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java

```diff
@@ -82,8 +82,9 @@
      * Check case-insensitive lookup
      */
     private static final String VALID_AUTHORITY = "https://login.windows.net/test.onmicrosoft.com";
-    private static final int ACTIVITY_TIME_OUT = 1000;
+    private static final int ACTIVITY_TIME_OUT = 2000;
```

Comment:
Because for the retry, it needs one second wait. So the time_out here should be bigger than 1s. This is the why I set it to 2s.